### PR TITLE
encode resource['url'] as unicode when logging

### PR DESCRIPTION
**Pull request by joetsoi**
if the resource url contains non ascii characters the datastore will fail with UnicodeValueError when attempting to write to the log

### CODE REVIEW

**Comment by joetsoi** [comment]
@domoritz just adding some tests.

File: ckanext/datastorer/commands.py [hunk 1]
```diff
@@ -117,12 +117,12 @@ def command(self):
...
                         logger.warn('Skipping resource %s from package %s '
                                 'because MIME type %s and format %s are '
-                                'unrecognized' % (resource['url'],
+                                'unrecognized' % (resource['url'].encode('utf-8'),
                                 package['name'], mimetype, resource['format']))
                         continue
 
                     logger.info('Datastore resource from resource %s from '
-                                'package %s' % (resource['url'],
+                                'package %s' % (resource['url'].encode('utf-8'),
```
Comment (domoritz) [inline ckanext/datastorer/commands.py]: I can't try it atm but does `u'package %s' % (resource['url'],` work as well? If yes, I'd prefer it and leave the encoding to the logger.

File: ckanext/datastorer/commands.py [hunk 2]
```diff
@@ -229,19 +229,19 @@ def command(self):
...
+                                              resource['format']))
                     continue
                 if (self.options.ignore and resource['id'] in
                         self.options.ignore):
                     logger.warn('Ignoring resource {0}'.format(resource['id']))
                     continue
                 logger.info('Datastore resource from resource {0} from '
-                            'package {0}'.format(resource['url'],
-                                                 package['name']))
+                    'package {0}'.format(resource['url'].encode('utf-8'),
```
Comment (domoritz) [inline ckanext/datastorer/commands.py]: Same here, can you try:

``` python
u'package {0}'.format(resource['url'], package['name']))
```

**Comment by joetsoi** [comment]
@domoritz the unicode strings work and are a much nicer looking solution

**Pull request merged**